src/report.rs

```diff
@@ -47,6 +50,55 @@ pub fn report_terminal(
     report_terminal_opts(r#match, code, path, &Opts::default(), writer)
 }
 
+fn highlight_code(code: &str, highlight_config: &HighlightConfiguration) -> String {
+    let mut highlighter = Highlighter::new();
+    let highlights = highlighter
+        .highlight(highlight_config, code.as_bytes(), None, |_| None)
```

Comment:
Please pass in a byte slice instead of crafting a string just to create a bytes array from it.